rpc/rpc-codegen/src/main/java/com/strategyobject/substrateclient/rpc/codegen/sections/RpcGeneratedSectionFactory.java

```diff
@@ -10,7 +10,10 @@
 import static com.strategyobject.substrateclient.rpc.codegen.sections.Constants.CLASS_NAME_TEMPLATE;
 
 public class RpcGeneratedSectionFactory {
```

Comment:
this class doesn't seem to be anyhow involved in code generation. What do you think about moving it to `rpc-sections` or `rpc-core`?